journal/modernized/32-epistles.adoc

```diff
@@ -115,10 +120,12 @@ and in the end produce both peace and comfort.
 My dear love is to you, as a true and faithful husband, which you may assure yourself of,
 wherever I am, or however I may be disposed of;
 and my tender love is to all honest-hearted Friends, as if I named them, and rest
+//after "rest" above, there's no punctuation in the PDF, is that okay?
 
-Your loving husband.
+[.signed-section-signature]
+Your loving husband
 
-=== Part of a Letter written by William Edmundson, at Barbadoes, to Friends in Ireland; dated the 8th of the First month, 1675.
+=== Part of a Letter written by William Edmundson, at Barbados, to Friends in Ireland; dated the 8th of the First month, 1675.
 
 My love is to all dear Friends.
```

Comment:
This is an unusual salutation, but it still seems to be serving that purpose in this letter. So I would label it [.salutation]